Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# SPDX-License-Identifier: curl
+
+# Self-contained build environment to match the release environment.
+#
+# Build with
+#
+#   docker build -t curl/curl .
+#
+# Then run commands from within the build environment, for example
+#
+#   docker run --rm -v $(pwd):/usr/src -w /usr/src curl/curl autoreconf -fi
+#   docker run --rm -v $(pwd):/usr/src -w /usr/src curl/curl ./configure --without-ssl --without-libpsl
+#   docker run --rm -v $(pwd):/usr/src -w /usr/src curl/curl make
+#   docker run --rm -v $(pwd):/usr/src -w /usr/src curl/curl ./maketgz 8.7.1
+
+FROM debian:bookworm-slim@sha256:993f5593466f84c9200e3e877ab5902dfc0e4a792f291c25c365dbe89833411f
```

Comment:
Forgive a docker newbie, but where does the data for this line come from?

Reply:
You can find it when you pull the image:
```
docker pull debian:bookworm-slim
bookworm-slim: Pulling from library/debian
8a1e25ce7c4f: Pull complete
Digest: sha256:ccb33c3ac5b02588fc1d9e4fc09b952e433d0c54d8618d0ee1afadf1f3cf2455
Status: Downloaded newer image for debian:bookworm-slim
docker.io/library/debian:bookworm-slim
```
But also probably using `docker manifest`.

Reply:
alternately one might be able to use the  existing curl-dev-debian image as a base (in curl ghcr)

Reply:
what's the practical difference ?

Reply:
I got this digest by going to

https://hub.docker.com/_/debian

checking tags

https://hub.docker.com/_/debian/tags

and filtering for bookworm-slim

---

I didn't know there is already a curl docker image; I read the original mail listing tools and operating system and simply went for debian bookworm. Happy to change this.